hw_8_Python/main.py

```diff
@@ -0,0 +1,219 @@
+        posts = []
+        for i in range(1, len(post)):
+            posts.append(post[i][post[i].find(",") + 1:post[i][post[i].find(",") + 1].find(',')])
+        file.close()
+
+    with open("files/Department.txt", encoding="utf-8") as file:
+        department = list(file.readlines())
+        departments = []
+        for i in range(1, len(department)):
+            departments.append(department[i][department[i].find(",") + 3:department[i].find("\n") - 1])
+        file.close()
+
+    with open("files/Employees.txt", "r", encoding="utf-8") as file:
+        flag = False
+        selected_string = list(file.readlines()[x].strip().split(", "))
+        print(selected_string)
+        selected_field = input("Что поменять: имя, дату рождения, телефон, должность?: ").strip().lower()
+        if selected_field == "имя":
+            selected_string[1] = input("Введите новое имя: ")
+            selected_string[1] = f'"{selected_string[1]}"'
+            print(selected_string)
+            rewrite(selected_string, x)
+        elif selected_field == "дату рождения":
+            selected_string[2] = input("Введите новую дату рождения: ")
+            selected_string[2] = f'"{selected_string[2]}"'
+            print(selected_string)
+            rewrite(selected_string, x)
+        elif selected_field == "телефон":
+            selected_string[3] = input("Введите новый телефон: ")
+            selected_string[3] = f'"{selected_string[3]}"'
+            print(selected_string)
+            rewrite(selected_string, x)
+        elif selected_field == "должность":
+            selected_string[4] = input("Введите новую должность: ").capitalize()
+            selected_string[4] = f'"{selected_string[4]}"'
+            for i in range(len(posts)):
+                if selected_string[4] in posts[i]:
+                    department_change = posts[i][len(posts[i]) - 2:len(posts[i]) - 1]
+                    selected_string[5] = departments[int(department_change)]
+                    flag = True
+
+            if flag == True:
+                print(selected_string)
+                rewrite(selected_string, x)
+
+            elif flag == False:
+                print("Такой должности не существует.")
+
+        else:
+            print("Названия команд: имя, дату рождения, телефон или должность введите правильно.")
+
+        file.close()
+
+
+def open_main_tab_in_notepad():
+    subprocess.Popen("notepad files/Employees.txt")
+    subprocess.Popen("notepad files/Post.txt")
+    subprocess.Popen("notepad files/Department.txt")
+
+
+def add_in_main_tab():
+    with open("files/Post.txt", encoding="utf-8") as file:
+        post = list(file.readlines())
+        posts = []
+        for i in range(1, len(post)):
+            posts.append(post[i][post[i].find(",") + 2:post[i][post[i].find(",") + 1].find(',')])
+        file.close()
+
+    with open("files/Department.txt", encoding="utf-8") as file:
+        department = list(file.readlines())
```

Comment:
```suggestion
        department = file.readlines()
```
Метод `split` итак возвращает список